packages/engine/LICENSE.md

```diff
@@ -276,7 +276,7 @@ license document, but changing it is not allowed.
   and must require no special password or key for unpacking, reading or
   copying.
 
-  7. Additional Terms.
+  ## 7. Additional Terms.
```

Comment:
Some code renders as code blocks because of leading spaces – see [preview](https://github.com/hashintel/hash/blob/3b8fa303fb0b9caead253d6bce76c61ce8d68eb1/packages/engine/LICENSE.md).

Markdownlint and Prettier are disabled for LICENSE.md files, so we don’t see any linting feedback. Happy to leave the files as is and revisit them later (formatting should affect the meaning). Alternatively, we can use `LICENSE.txt` if we don’t want to mess with md.

Reply:
I'd prefer they be in markdown for readability. If we can fix their appearing in codeblocks through linting then I'm open to it, so long as we're not changing the words of the text or altering meaning in any way (e.g. ensuring clauses remain nested properly).

Reply:
How about creating a follow-up task to clean-up all license files? I guess our immedeate goal is to communicate meaning (which this PR does).

Reply:
Captured at https://app.asana.com/0/1200211978612931/1201648093145571/f (internal HASH company link)